lib/index.js

```diff
@@ -128,18 +134,25 @@ util.inherits(SpawnMocha, EventEmitter);
 
 var mochaStream = function mocha(opts) {
   opts = opts || {};
+  var files = [];
```

Comment:
Thanks for taking the time to look at this. I apologize that I'm learning as I go here. I'll give
what you've said some thought and try the promises approach.